README.md

```diff
@@ -2,21 +2,117 @@
 [![Build Status](https://travis-ci.org/thoughtbot/french_toast.svg?branch=master)](https://travis-ci.org/thoughtbot/french_toast)
 [![Code Climate](https://codeclimate.com/github/thoughtbot/french_toast/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/french_toast)
 
-Communicate information about background jobs.
+Communicate information about background jobs in Rails seamlessly.
+
+FrenchToast attempts to streamline some [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement) by providing a simple and opinionated interface. In order to best take advantage of this, it is important to render that information on the backend. This allows FrenchToast to figure out the best possible communication stream, by starting at the most enhanced experience and falling back to the most basic.
+
+With `FrenchToast` it will:
+
+1. Communicate through websockets
+1. If that is unavailable, communicate on next page refresh
+1. If the user hasn't seen the content, send them an email
 
 ## Installation
 
-1. Add the French Toast gem to your Gemfile:
+1. Add the `FrenchToast` gem to your Gemfile:
 
   ```
   gem "french_toast"
   ```
 
 1. Run `bundle install` from your shell.
```

Comment:
This is what I see in the README on master now:
![screen shot 2017-08-10 at 1 04 53 pm](https://user-images.githubusercontent.com/1369728/29182373-a03b4f2e-7dcc-11e7-8c74-4470d42821f6.png)